iothub/service/src/AmqpServiceClient.cs

```diff
@@ -50,6 +52,9 @@ public AmqpServiceClient(IotHubConnectionString iotHubConnectionString, bool use
                 ExceptionHandlingHelper.GetDefaultErrorMapping(),
```

Comment:
nit: strange indentation here

Reply:
these are args for the constructor, hence the extra tab :)